redcap_bridge/test_redcap/test_server_elab_interface.py

```diff
@@ -0,0 +1,31 @@
+from redcap_bridge.server_elab_interface import (upload_template_from_csv, download_experiment,
+                                                 upload_template)
+from redcap_bridge.test_redcap.test_utils import (test_directory, initialize_test_dir)
+
+SERVER_CONFIG_YAML = (test_directory / 'testfiles' / 'TestProject' / 'project.json').resolve()
+
+test_file = (test_directory / 'testfiles' / 'elabConversion' / 'csvRecord.csv')
+
+
+def test_create_template(initialize_test_dir):
```

Comment:
Hi @killianrochet You are still mixing `create` and `upload` here in the name of the test function versus the name of the function you test.

Reply:
I have done a commit to fix this